cmd/benchttp/run.go

```diff
@@ -0,0 +1,151 @@
+package main
+
+import (
+	"bufio"
+	"context"
+	"errors"
+	"flag"
+	"fmt"
+	"os"
+	"os/signal"
+
+	"github.com/benchttp/runner/config"
+	"github.com/benchttp/runner/internal/configfile"
+	"github.com/benchttp/runner/internal/configflags"
+	"github.com/benchttp/runner/output"
+	"github.com/benchttp/runner/requester"
+)
+
+var defaultConfigFiles = []string{
```

Comment:
See https://github.com/benchttp/runner/pull/35#discussion_r815689476.

Your description is deliberately convoluted, but in better words we simply inject a value in an independent structure. That value happens to be available in the global scope: it could not be as well, or change at runtime without affecting `cmdRun`.